lib/segment/src/index/hnsw_index/gpu/gpu_vector_storage/tests.rs

```diff
@@ -0,0 +1,678 @@
+use std::path::Path;
```

Comment:
A file with all unit tests: f32, f16, u8 dense/multivectors with SQ, BQ, PQ.

Reply:
Maybe we can add this as a comment to the module.

```suggestion
//! Unit tests for GPU vector storage, covering f32, f16, u8 dense/multivectors with SQ, BQ, PQ.

use std::path::Path;
```